test/user.test.js

```diff
@@ -1763,6 +1763,367 @@ describe('User', function() {
     });
   });
 
+  describe('Email Update', function() {
+    describe('User changing email property', function() {
+      var usersId;
+      var currentEmailCredentials = { email: 'm@c.com', password: 'bar' };
+      var updatedEmailCredentials = { email: 'n@d.com', password: 'bar' };
+
+      beforeEach(function(done) {
+        async.series([
+          function(next) {
+            User.create({ email: currentEmailCredentials.email,
+            password: currentEmailCredentials.password }, function(err, user) {
+              usersId = user.id;
+              next();
+            });
+          },
+          function(next) {
+            User.login({ email: currentEmailCredentials.email,
+            password: currentEmailCredentials.password }, function(err, accessToken1) {
+              if (err) return next(err);
+              assert(accessToken1.userId);
+              next();
+            });
+          },
+          function(next) {
+            User.login({ email: currentEmailCredentials.email,
+            password: currentEmailCredentials.password }, function(err, accessToken2) {
+              if (err) return next(err);
+              assert(accessToken2.userId);
+              next();
+            });
+          },
+        ], function(err) {
+          if (err) return done(err);
+          done();
+        });
+      });
+
+      afterEach(function(done) {
+        User.deleteById(usersId, function(err, user) {
+          if (err) return done (err);
+          done();
+        });
+      });
+
+      it('invalidates sessions once the email is changed using `updateAttribute`', function(done) {
```

Comment:
Use `when` instead of `once`:

`invalidates sessions when the email...`

Please apply to all other new tests too.